scripts/lib/wic/canned-wks/rpi4-sdimg.wks

```diff
@@ -0,0 +1,5 @@
+part /boot --source bootimg-partition --ondisk mmcblk1 --fstype=vfat --label boot --active --align 4096 --size 10
+
+part / --source rootfs --ondisk mmcblk1 --fstype=ext4 --mkfs-extraopts "-T default" --label root --align 4096 --exclude-path=boot --use-uuid
```

Comment:
I'm wondering if the both `--ondisk` and `--use-uuid` options are needed at the same time. With uuid specification, we have not had to use `mmcblkX` that may be changed due to the identification order. The advantage to that would be lost if --ondisk is also used.

Here's the description of the part command.

https://docs.yoctoproject.org/4.2.3/ref-manual/kickstart.html#command-part-or-partition

![image](https://github.com/miraclelinux/meta-emlinux/assets/135098796/3e4f716b-ecfc-425b-a560-dda64e845582)

Is it enough to set either option to boot the board?

Reply:
RPi4 image successfully booted without `--ondisk` as you wrote.
Therefore, I removed `--ondisk` option from the file.